install/installer/pkg/config/v1/envvars.go

```diff
@@ -418,7 +418,7 @@ func (v version) BuildFromEnvvars(in interface{}) error {
 		cfg.Experimental.Telemetry.Data.Platform = telemetryValue
 	}
 
-	if cfgPatch := envvars.ConfigPatch; cfgPatch != "" {
+	if cfgPatch := envvars.ConfigPatch; envvars.AdvancedModeEnabled && cfgPatch != "" {
```

Comment:
nit: I understand that the `ConfigPatch` has to be the last change to be applied (as the expectation here is that it can override anything). How about we [move the advanced mode toggle](https://github.com/gitpod-io/gitpod/blob/80d98f25fcde9e8629cb4d157bf3232a161964eb/install/installer/pkg/config/v1/envvars.go#L375) a bit below here so that we can have all the advanced mode settings at the same place? 🤔

Reply:
You know what, you're right. I was 50/50 when I first did it (mainly because it makes the lines of code changed much larger) - your suggestion pushes me over 50/50 into one option